pkg/util/file_indexer.go

```diff
@@ -149,17 +146,9 @@ func Run(directory string, ignoreRules []string) (filesChanged []string, filesDe
 // filepath is the location of the file to which it is supposed to be written
 func write(filePath string, writeMap map[string]FileData) error {
 	jsonData, err := json.Marshal(writeMap)
```

Comment:
two `err` one after the other, but error checked in the end. which means this error will be ignore